### DISCUSSION TITLE
How to add aligned icons to "Layout Header and Footer" example with qtabs ?

### DISCUSSION THREAD

**Discussion by ZheFrench** [quasarframework/quasar#15228] [General - Components / Directives / etc] [answered]
There is nice display [here](https://quasar.dev/layout/header-and-footer) section Playing with Qtabs I would like to reproduce with icons. I would like to add icons next to each  menu items(q-route-tab-to)
I'm struggling to do this nicely. Could you give a hand on this and provide an example ? Tks.

`  <q-layout  view="hHh lpr fFf" class="fullscreen" style="overflow: auto;" >
    
    <q-header elevated class="bg-primary text-white" height-hint="98">
      <q-toolbar>
        <q-toolbar-title>
          <q-avatar>
            <img src="/favicon_io/favicon-32x32.png">
          </q-avatar>
          LRdb : Database for ligand-receptor interactions and their associated complexes.
        </q-toolbar-title>
      </q-toolbar>

      <q-tabs align="left"  indicator-color="yellow" >
         <q-icon name="home"  size="sm" /><q-route-tab to="/" label="Home" /> 
        <q-route-tab to="/search" label="Search" />
        <q-route-tab to="/download" label="Download" />
        <q-route-tab to="/submit" label="Submit" />
        <q-route-tab to="/api" label="API Documentation" />
        <q-route-tab to="/analysis" label="Analysis" />
        <q-route-tab to="/faq" label="FAQ" />
      </q-tabs>
    </q-header>

    <q-page-container>
      <router-view />
    </q-page-container>

    <q-footer elevated class="bg-grey-8 text-white">
      
      </q-toolbar>
    </q-footer>
`

**Comment by suleimanas** [comment, marked as answer]
QRouteTab has an **icon** prop, and QTab has **inline-label** this is what you want
```
<q-tabs align="left"  indicator-color="yellow" inline-label>
    <q-route-tab to="/" label="Home" icon="home" />
</q-tabs>
```

https://quasar.dev/vue-components/tabs#qroutetab-api

**Comment by ZheFrench** [comment]
Yeah thanks , didn't see that, but the home icon is displaying at the top to the text , so was wondering how to put it next to it.

Ah ok just saw your edit :/

Ok thanks again, that was super fast ! this is awesome !

**Reply by suleimanas** [reply]
QTab has inline-label prop, Updated the answer
